client/landing/stepper/declarative-flow/internals/components/help-center/help-center-action-button.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useTranslate } from 'i18n-calypso';
+import type { HelpCenterSelect } from '@automattic/data-stores';
+
+import './help-center-action-button.scss';
+
+export const HelpCenterActionButton = () => {
```

Comment:
A new component to be easier to add the Help Center to the steps by passing this to the `customizedActionButtons` prop of the `StepContainer` component.